site/docs/try/releases.md

```diff
@@ -1,5 +1,11 @@
 # Releases
 
+## 0.10.0 Release (November 12, 2021)
```

Comment:
release id is wrong !!

Reply:
I took a liberty to add comment as github suggestion and applied it directly from UI by doing commit suggestion !!